src/sidebar/components/MarkdownEditor.js

```diff
@@ -138,20 +144,52 @@ function ToolbarButton({
 
   if (label) {
     return (
-      <LabeledButton classes="font-normal TransparentButton" {...buttonProps}>
+      <LabeledButton
+        classes={classNames(
+          'font-normal bg-transparent',
+          // TODO: Refactor shared button styles to reduce specificity and make
+          // this !important rule unnecessary
+          'hover:!bg-transparent'
+        )}
+        {...buttonProps}
+      >
         {label}
       </LabeledButton>
     );
   }
   return (
-    <IconButton classes="MarkdownEditor__toolbar-button" {...buttonProps} />
+    <IconButton
+      classes="py-3.5 px-2 text-tiny touch:text-base"
+      {...buttonProps}
+    />
   );
 }
 
 /**
  * @typedef {'bold'|'italic'|'quote'|'link'|'image'|'math'|'numlist'|'list'|'preview'|'help'} ButtonID
  */
 
+/**
+ * @typedef {import('preact').Ref<HTMLTextAreaElement>} TextAreaRef
+ * @typedef {import('preact').JSX.HTMLAttributes<HTMLTextAreaElement>} TextAreaAttributes
+ *
+ * @param {TextAreaAttributes & { classes?: string, containerRef?: TextAreaRef }} props
+ */
+function TextArea({ classes, containerRef, ...restProps }) {
+  return (
+    <textarea
+      className={classNames(
+        'border rounded-sm p-2',
+        'text-color-text-light bg-grey-0',
+        'focus:bg-white focus:outline-none focus:shadow-focus-inner',
```

Comment:
Is this combination of styles the same as for other input fields? Is that worth extracting into a pattern?

Reply:
Almost certainly, as it is based on a SASS mixin implementation. It's on my radar.